### PR TITLE
Added config to jQueryAdapter for mapping dataType to http method

### PR DESCRIPTION
**Pull request by alexnsolo**

### CODE REVIEW

**Comment by stoodder** [comment]
Looks good!

**Comment by stoodder** [comment]
The last thing I'd have you do is also to add this to the jquery_adapter unit tests

File: coffee/adapters/jquery_adapter.coffee [hunk 1]
```diff
@@ -26,7 +27,11 @@ class @jQueryAdapter extends Falcon.Adapter
 	standardizeOptions: ( data_object, type, options, context ) ->
 		output_options = super( data_object, type, options, context )
 
-		output_options.dataType = "json" unless isString(output_options.dataType)
+		if @config.dataTypeMap.hasOwnProperty(type)
+			output_options.dataType = @config.dataTypeMap[type]
+		else
+			output_options.dataType = "json" unless isString(output_options.dataType)
+
```
Comment (joshdholtz) [inline coffee/adapters/jquery_adapter.coffee:35]: @civilframe Can you also add an `#END if` here? Just way so we can visualize were the if statement ends :grin:

**Comment by alexnsolo** [comment]
Thanks for the feedback. Let me know if there's anything else I can tweak.

**Pull request merged**